pubsub.go

```diff
@@ -734,10 +736,11 @@ func (p *PubSub) GetTopics() []string {
 // Publish publishes data to the given topic.
 func (p *PubSub) Publish(topic string, data []byte) error {
 	seqno := p.nextSeqno()
+	hostID := p.host.ID()
```

Comment:
nit: just `id` is fine. `hostID` is kinda foreign, as hosts don't have IDs, but rather peers do.